src/tennisSimulator/MatchParticipants.java

```diff
@@ -0,0 +1,25 @@
+package tennisSimulator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MatchParticipants {
+
+    private final List<Player> participants;
+
+    public MatchParticipants() {
+        participants = new ArrayList<>();
+    }
+
+    public void addMatchParticipant(Player player) {
+        if (participants.size() <= 2) {
```

Comment:
In this case you are allowing 3rd player to be added.

[Optional]: Also wonder if you really need a list here. Later on when you will be fetching players from this class code will look something like `matchParticipants.getParticipants().get(0) `instead of `matchParticipants.getFirstPlayer();`